internal/test/network/client.go

```diff
@@ -4,29 +4,29 @@ import (
 	"context"
 
 	"github.com/docker/docker/api/types"
-	"github.com/docker/docker/api/types/filters"
-	"github.com/docker/docker/api/types/network"
+	"github.com/docker/docker/client"
 )
 
 // FakeClient is a fake NetworkAPIClient
 type FakeClient struct {
+	client.NetworkAPIClient
 	NetworkInspectFunc func(ctx context.Context, networkID string, options types.NetworkInspectOptions) (types.NetworkResource, error)
 }
 
-// NetworkConnect fakes connecting to a network
-func (c *FakeClient) NetworkConnect(ctx context.Context, networkID, container string, config *network.EndpointSettings) error {
-	return nil
-}
-
-// NetworkCreate fakes creating a network
-func (c *FakeClient) NetworkCreate(_ context.Context, _ string, options types.NetworkCreate) (types.NetworkCreateResponse, error) {
-	return types.NetworkCreateResponse{}, nil
-}
-
-// NetworkDisconnect fakes disconnecting from a network
-func (c *FakeClient) NetworkDisconnect(ctx context.Context, networkID, container string, force bool) error {
-	return nil
-}
+// // NetworkConnect fakes connecting to a network
+// func (c *FakeClient) NetworkConnect(context.Context, string, string, *network.EndpointSettings) error {
+// 	return nil
+// }
+//
```

Comment:
Looks like these "fakes" don't need all these "unimplemented" methods; instead of changing all the unused arguments, we may be able to remove the stubs and just embed the interface